### ISSUE TITLE
[Accessibility]: Dashboard - Create insight doesn't show created insight on the dashboard page

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#41805]
### Audit type

General/Other

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33493

### Problem description

After creating an insight, the insight is appended to the bottom on the dashboard page. In case if dashboard already has enough insights, this happens offscreen. There is no "success" indication that the insight actually was created.

### Expected behavior

Related to https://github.com/sourcegraph/sourcegraph/issues/41765
Either some "toast" alerts the user of the insight created, or scroll to the bottom to see the added insight.

We already have logic about scrolling so I think this would be a good first step here to solve this problem.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @joelkw @felixfbecker @vovakulikov

**Comment by felixfbecker** [comment]
Isn't this a duplicate of https://github.com/sourcegraph/sourcegraph/issues/41765?

**Comment by vovakulikov** [comment]
I think so, closing in favour of https://github.com/sourcegraph/sourcegraph/issues/41765
